src/main/java/dev/jbang/source/TagReader.java

```diff
@@ -153,6 +155,27 @@ protected boolean isDescriptionDeclare(String line) {
 		return line.startsWith(DESCRIPTION_COMMENT_PREFIX);
 	}
 
+	public Optional<ResourceRef> getDocs(ResourceResolver resolver) {
+		return getTags()
+			.filter(this::isDocsDeclare)
+			.map(s -> s.substring(DOCS_COMMENT_PREFIX.length()))
+			.map(TagReader::createResourceReference)
+			.findFirst();
+	}
+
+	private static ResourceRef createResourceReference(String s) {
```

Comment:
No, this isn't how it's supposed to be done, you _have_ to use the resolver. Like I said before, don't worry about things being downloaded, that's something we can fix later.